pandas/computation/pytables.py

```diff
@@ -188,10 +188,6 @@ def stringify(value):
             if v.tz is not None:
                 v = v.tz_convert('UTC')
             return TermValue(v, v.value, kind)
-        elif (isinstance(v, datetime) or hasattr(v, 'timetuple') or
```

Comment:
this was some really old compat IIRC. (or maybe just copied it from somewhere :>)

improving test coverage by deletions!